src/engine/TPA.cc

```diff
@@ -1515,6 +1516,8 @@ class TransitionSystemNetworkManager {
 
     InvalidityWitness computeInvalidityWitness() const;
 
+    ValidityWitness computeValidityWitness();
```

Comment:
Can it be `const`?

Reply:
Unfortunately no, because `queryTransitionSystem` is called inside this function